### PR TITLE
Get master tags

### PR DESCRIPTION
**Pull request by jyasveer**

### CODE REVIEW

**Comment by jpopelka** [comment]
Could you get rid of all the *"fix linting errors"*, *"add a line"* commits ?
`git rebase -i master` (`fixup` command) is your friend.

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -462,23 +463,25 @@ def post():
...
-            return s3.store_in_bucket(input_json)
+            return s3.store_master_tags(input_json)
         else:
             if 'user' not in input_json:
                 raise HTTPError(400, error="Expected user name in the request")
 
             if 'data' not in input_json:
                 raise HTTPError(400, error="Expected tags in the request")
 
+            init_selinon()
```
Comment (fridex) [inline bayesian/api_v1.py]: It would be nice to have a decorator for this so this is not spread across sources randomly.

Comment (fridex) [reply bayesian/api_v1.py]: (but it is not requirement from my side)

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
                               get_component_percentile_rank, usage_rank2str,
                               MavenCoordinates, case_sensitivity_transform)
 from f8a_worker.manifests import get_manifest_descriptor_by_filename
+from f8a_worker.setup_celery import init_selinon
```
Comment (msrb) [inline bayesian/api_v1.py]: I think this shouldn't be needed any more. We initialize selinon only once in https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/__init__.py#L85.

File: bayesian/api_v1.py [hunk 3]
```diff
@@ -509,6 +513,28 @@ def get(user, ecosystem):
...
 
+class MasterTagsGET(ResourceWithSchema):
+    method_decorators = [login_required]
+
+    @staticmethod
+    def get(ecosystem):
+        if not ecosystem:
+            raise HTTPError(400, error="Expected ecosystem in the request")
+
+        init_selinon()
```
Comment (msrb) [inline bayesian/api_v1.py]: Not needed.

**Comment by jyasveer** [comment]
[test]

**Comment by jyasveer** [comment]
[test]

**Comment by msrb** [comment]
@jyasveer worker PR is merged, last missing piece here should be to update worker version in Dockerfile: https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/Dockerfile#L4

Should be `656d3e7`.

**Comment by jyasveer** [comment]
@msrb thanks for merging the worker PR. I have updated the commit for worker in server Dockerfile. Could you also merge this PR now ?

**Comment by msrb** [comment]
Will do 😉  Just waiting for CI to finish.

**Pull request merged**